docusaurus/docs/operate/quickstart/supplier_cheatsheet.md

```diff
@@ -0,0 +1,265 @@
+:::tip
+
+You can set the `--keyring-backend` flag to `test` to avoid entering the password
+each time.
+
+Learn more about [cosmos keyring backends here](https://docs.cosmos.network/v0.46/run-node/keyring.html).
+
+:::
+
+### Prepare your environment
+
+For convenience, we're setting several environment variables to streamline
+the process of interacting with the Shannon network:
+
+We recommend you put these in your `~/.bashrc` file:
+
+```bash
+export NODE="https://shannon-testnet-grove-rpc.beta.poktroll.com"
+export NODE_FLAGS="--node=https://shannon-testnet-grove-rpc.beta.poktroll.com"
+export TX_PARAM_FLAGS="--gas=auto --gas-prices=1upokt --gas-adjustment=1.5 --chain-id=pocket-beta --yes"
+export SUPPLIER_ADDR=$(poktrolld keys show supplier -a)
+
+# Optionally, to avoid entering the password each time:
+# export SUPPLIER_ADDR=$(poktrolld keys show supplier -a --keyring-backend test
+```
+
+:::tip
+
+You can put the above in a special `~/.poktrollrc` and add `source ~/.poktrollrc` to
+your `~/.profile` file for a cleaner organization.
+
+:::
+
+## Supplier Configuration
+
+### Fund the Supplier account
+
+Run the following command to get the `Supplier`:
+
+```bash
+echo "Supplier address: $SUPPLIER_ADDR"
+```
+
+Then use the [Shannon Beta TestNet faucet](https://faucet.beta.testnet.pokt.network/) to fund the account.
+
+Afterwards, you can query the balance using the following command:
+
+```bash
+poktrolld query bank balances $SUPPLIER_ADDR $NODE_FLAGS
+```
+
+:::tip
+
+You can find all the explorers, faucets and tools at the [tools page](../../explore/tools.md).
+
+:::
+
+### Stake the Supplier
+
+:::info
+
+For an in-depth look at how to stake a supplier, see the [Supplier configuration docs](./../configs/supplier_staking_config.md).
+
+The example below is a very quick and simple way to get you started by staking for
+Pocket Network's Morse service on Shannon using a public RPC endpoint provided by
+[Liquify](https://liquify.com/).
+
+:::
+
+Retrieve your external IP address:
+
+```bash
+EXTERNAL_IP=$(curl -4 ifconfig.me/ip)
+```
+
+Choose a port that'll be publicly accessible from the internet (e.g. `8545`)
+
+```bash
+sudo ufw allow 8545/tcp
+```
+
+Create a Supplier stake configuration file:
+
+```bash
+cat <<EOF > /tmp/stake_supplier_config.yaml
+owner_address: $SUPPLIER_ADDR
+operator_address: $SUPPLIER_ADDR
+stake_amount: 1000069upokt
+default_rev_share_percent:
+  $SUPPLIER_ADDR: 100
+services:
+  - service_id: "morse"
+    endpoints:
+      - publicly_exposed_url: http://$EXTERNAL_IP:8545
+        rpc_type: JSON_RPC
+EOF
+```
+
+And run the following command to stake the `Supplier`:
+
+```bash
+poktrolld tx supplier stake-supplier --config /tmp/stake_supplier_config.yaml --from=$SUPPLIER_ADDR $TX_PARAM_FLAGS $NODE_FLAGS
+
+# Optionally, to avoid entering the password each time:
+# poktrolld tx supplier stake-supplier --config /tmp/stake_supplier_config.yaml --from=$SUPPLIER_ADDR $TX_PARAM_FLAGS $NODE_FLAGS --keyring-backend test
```

Comment:
(same feedback as above)

```suggestion
# Optionally, to avoid entering the password each time:
# poktrolld tx supplier stake-supplier --config /tmp/stake_supplier_config.yaml --from=$SUPPLIER_ADDR $TX_PARAM_FLAGS --keyring-backend test
```

Reply:
Resolving as wont-do and delegating to #1011